cmd/config/subcommand/project/project_config.go

```diff
@@ -45,8 +46,9 @@ var DefaultProjectConfig = &ConfigProject{
 }
 
 // GetProjectSpec return project spec from a file/flags
-func (c *ConfigProject) GetProjectSpec(id string) (*admin.Project, error) {
+func (c *ConfigProject) GetProjectSpec() (*admin.Project, error) {
 	projectSpec := admin.Project{}
+	id := config.GetConfig().Project
```

Comment:
nit: can you define this closest to where it's used?

Reply:
Also I think we should use one consistent way to set the project id flag, rather than using the global project flag here but the id flag in else block below, can we exclusively read the `ConfigProject` id field instead? And either error  if the global project flag is set too (or fall back to it when the `ConfigProject` ID isn't set)

Reply:
I think it already behaves as you described, it only uses global project flag if ConfigProject ID isn't set